01-atoms/links/links.stories.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import Link from './Link.component';
+
+/**
+ * Storybook Definition.
+ */
+export default { title: 'Atoms/Link' };
+
+export const link = () => (
+  <Link 
+    children="This is my link text" 
+    linkURL="https://github.com/emulsify-ds/emulsify-drupal"
+    linkTarget="_blank"
+    linkTitleText="This is my title text" 
+  />
+);
```

Comment:
Consider adding a new line at the end of the file. I'm just being old school, not sure that it matters.